### ISSUE TITLE
Are you using only the reconstruction loss during your training process?

### ISSUE THREAD

**Issue by matri123** [IDKiro/CBDNet-tensorflow#3]
In your training code, only the reconstruction loss is used. There are two loss functions for the noise map in the original map. Will you add them later? And, during the training process, the author said that the network is alternately trained using real noise images and synthetic noise images. Will you add them?

**Comment by IDKiro** [comment]
sorry, i forgot modifing the loss function, i'll achieve it later.
real noise images will be added too.
